generate-wasm-examples/generate_wasm_examples.sh

```diff
@@ -14,6 +17,24 @@ sed -i.bak 's/canvas: None,/canvas: Some("#bevy".to_string()),/' crates/bevy_win
 # setting the asset folder root to the root url of this domain
 sed -i.bak 's/asset_folder: "assets"/asset_folder: "\/assets\/examples\/"/' crates/bevy_asset/src/lib.rs
 
+export CARGO_TARGET_DIR="target"
```

Comment:
Why is this an export?

Reply:
Actually, is this used at all?

Reply:
It's actually the default.

This shouldn't  affect the caller's environment (it only affects programs called within the script, just tried it with with a `test.sh`, content of which was `export FOO="bar"`).

The default value of `CARGO_TARGET_DIR` is indeed `"target"`, on my machine I set it to something else, but it breaks this script. So I set the value to the default one here, so that when `cargo` is called in this script, it uses a consistent target dir.

Reply:
I guess it doesn't break anything to leave it there, but it feels a bit weird since in CI it should already be the default. I guess it makes it less likely to break unexpectedly.